libindy-crypto/src/cl/issuer.rs

```diff
@@ -189,17 +189,17 @@ impl Issuer {
         trace!("Issuer::sign_claim: >>> prover_id: {:?}, blinded_ms: {:?}, claim_values: {:?}, issuer_pub_key: {:?}, issuer_priv_key: {:?}, rev_idx: {:?}, \
         rev_reg_pub: {:?}, rev_reg_priv: {:?}", prover_id, blinded_ms, claim_values, issuer_pub_key, issuer_priv_key, rev_idx, rev_reg_pub, rev_reg_priv);
 
-        let m_2 = Issuer::_calc_m2(prover_id, rev_idx)?;
+        let context = Issuer::_gen_context(prover_id, rev_idx)?;
```

Comment:
Current names of variables are follow to CL math paper. The only way to understand code properly is reading this paper first. Not sure that providing our own names are better way for understanding.

@ashcherbakov What do you think about this?

Reply:
Yes, the paper is a required read, but it is better to keep meaningful variable names in you head than not

Reply:
If we want to give a more meaningful name for `m2`, then probably we should call it `context_attr`, not just `context`.
Also we must put a comment that `context_attr` == `m2` in the paper.